LOAN/LoanContracts/LoanApplications.rdf

```diff
@@ -157,14 +368,200 @@
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-ln-app;PreApprovalRequest">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-com;Request"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;hasBorrower"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;Borrower"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-app;hasRequestedAmount"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>pre-approval request</rdfs:label>
+		<skos:definition>request from a potential borrower that a lender commit to pre-approving the borrower for a loan of up to a specified amount of money</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>This may also include limits on the region where to purchase.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-ln-app;PublicRecord">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-doc;Record"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+				<owl:onClass rdf:resource="&fibo-fbc-pas-caa;Account"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;isEvidencedBy"/>
+				<owl:onClass rdf:resource="&fibo-fnd-arr-doc;LegalDocument"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-cr;isClassifiedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;PublicRecordCategory"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>public record</rdfs:label>
+		<skos:definition>record about an action involving a party that is publicly available from a court or other government agency</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>This can include court actions such as bankruptcy and foreclosure, as well as liens and other events that have been recorded.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-ln-app;PublicRecordCategory">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
+		<rdfs:label>public record category</rdfs:label>
+		<skos:definition>classifier of public records relevant to a loan application, e.g. tax lien, wage garnishment, foreclosure</skos:definition>
+	</owl:Class>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-app;PublicRecordCategory-BankruptcyRecord">
+		<rdf:type rdf:resource="&fibo-loan-ln-app;PublicRecordCategory"/>
+		<rdfs:label>bankruptcy record</rdfs:label>
```

Comment:
Agreed - again all of this was in LoanCore, and I moved it here so that we could actually think about getting an FCT up and running starting with LoanCore.  I didn't attempt to address the content.